src/inspector_agent.cc

```diff
@@ -170,24 +173,28 @@ class AgentImpl {
   void WaitForDisconnect();
 
  private:
+  using MessageQueue =
+    std::vector<std::pair<int, const blink::protocol::String16>>;
```

Comment:
Done. String16STD.h already imports it into the default namespace - so I'll be using it.